Gordon360/Services/HousingService.cs

```diff
@@ -376,27 +370,25 @@ public int EditApplication(string username, string sess_cde, int applicationID,
                 }
                 sessionParam = new SqlParameter("@SESS_CDE", sess_cde);
 
-                applicantResult = RawSqlQuery<AA_Applicants>.query("UPDATE_AA_APPLICANT @APPLICATION_ID, @ID_NUM, @APRT_PROGRAM, @SESS_CDE", appIDParam, idParam, programParam, sessionParam); //run stored procedure
+                int? applicantResult = context.Database.ExecuteSqlCommand("UPDATE_AA_APPLICANT @APPLICATION_ID, @ID_NUM, @APRT_PROGRAM, @SESS_CDE", appIDParam, userParam, programParam, sessionParam); //run stored procedure
                 if (applicantResult == null)
                 {
-                    throw new ResourceCreationException() { ExceptionMessage = "Applicant with ID " + applicant.StudentID + " could not be updated." };
+                    throw new ResourceCreationException() { ExceptionMessage = "Applicant " + applicant.StudentID + " could not be updated." };
```

Comment:
We could, though it seems unnecessary since this is an error message that only the devs can ever see, and its purpose/meaning is still clearly communicated, and we plan to change it back to username in the future anyway.

Reply:
Okay, that's fine then :)